docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_independent_evaluation.yml

```diff
@@ -216,8 +217,10 @@ subquestion: |
   % endif
 fields:
   - Title: evaluators[i].name.title
-  - code: |
-      evaluators[i].name_fields()
+  - First name: evaluators[i].name.first
```

Comment:
See better way to do suffix above https://suffolklitlab.org/docassemble-AssemblyLine-documentation/docs/question_library/names#optional-omit-suffix